_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -4006,6 +4006,7 @@ mfxStatus VideoVPPHW::SyncTaskSubmission(DdiTask* pTask)
         return sts;
     }
 
+#if !defined(MFX_VA_LINUX)
```

Comment:
I restrict the code to work on non-Linux instead of Linux, hence on non-Linux, it works as before. On Linux, we may use the VPP features provided by media driver.

Reply:
BTW maybe we may utilize the driver to enable mirroring on non-Linux too, but I haven't the environment to test it on non-Linux, so I restricted my patch on Linux only and don't change any functionality on non-Linux platforms.